scripts/shared/release.sh

```diff
@@ -24,14 +24,16 @@ source ${SCRIPTS_DIR}/lib/version
 
 function release_image() {
     local image=$1
-    local images=("${image}:${TRAVIS_COMMIT:0:7}" "${image}:${release_tag#v}")
+    local images=("${image}:${commit_hash:0:7}" "${image}:${release_tag#v}")
 
     for target_image in "${images[@]}"; do
         docker tag ${image}:${VERSION} ${target_image}
         docker push ${target_image}
     done
 }
 
+commit_hash=${GITHUB_SHA:-${TRAVIS_COMMIT}}
```

Comment:
haha, makes sense to keep compatibility with both :)

Reply:
For now, until we get rid of travis :)